packages/next/src/build/webpack-config.ts

```diff
@@ -245,6 +245,7 @@ export function getDefineEnv({
             process.env.NEXT_EDGE_RUNTIME_PROVIDER || 'edge-runtime'
           ),
         }),
+    'process.turbopack': JSON.stringify(false),
```

Comment:
why is that needed in this PR?

Reply:
`image-response` is using `process.turbopack` to display an error, try to optimize it as much as possible